install/kubernetes/cilium/charts/config/templates/configmap.yaml

```diff
@@ -134,6 +134,12 @@ data:
   # Only effective when monitor aggregation is set to "medium" or higher.
   monitor-aggregation-flags: {{ .Values.global.bpf.monitorFlags }}
 
+{{- if .Values.global.cni.customConf }}
```

Comment:
I don't understand how blacklisting conflicting routes relates to the customConf option. Typically the customConf option is used when Cilium is chaining on top of another CNI, in which case it shouldn't be handling IPAM in general anyway. Does this mitigate a specific issue you've observed? Can you provide details?

Reply:
From initial review I would expect that this block is not conditional and the actual setting is deferred to the user rather than configuring a different default in this chart depending on other options.

Reply:
Hey, you are right, it was a mistake from my side. In the last commit, I have removed the conditional part here.